Tests/ML/test_download_upload.py

```diff
@@ -270,6 +243,8 @@ def test_mount_or_download(test_output_dirs: OutputFolderForTests) -> None:
         if not is_lightning_model:
             assert container.config.local_dataset == container.local_dataset
 
+        # With runs outside of AzureML, a local dataset should be used as-is. Azure dataset ID is ignored here.
+        shutil.copy(full_ml_test_data_path(DATASET_CSV_FILE_NAME), root / DATASET_CSV_FILE_NAME)
```

Comment:
why is this still needed? I thought your code is already copying everything in the _test_... method?